pkg/core/utils_test.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package core_test
+
+import "flag"
+
+var update = flag.Bool("update", false, "update .golden files")
```

Comment:
Is there another definition of this flag (in collector) that needs to be removed?

Reply:
Those are different packages (`collector_test` vs `core_test`), so the flag needs to be defined in both places.